src/psyclone/tests/psyir/transformations/loop_fusion_test.py

```diff
@@ -640,31 +640,8 @@ def test_fuse_no_symbol(fortran_reader, fortran_writer):
     enddo
   enddo""" in out
 
```

Comment:
Please update the docstring for this test as we no longer have to patch the symbol table.

Reply:
Done.